tests/unit_tests/pandas_postprocessing/test_rolling.py

```diff
@@ -162,8 +162,8 @@ def test_rolling_after_pivot_with_single_metric():
         pd.DataFrame(
             data={
                 "dttm": pd.to_datetime(["2019-01-01", "2019-01-02"]),
-                FLAT_COLUMN_SEPARATOR.join(["sum_metric", "UK"]): [5.0, 12.0],
-                FLAT_COLUMN_SEPARATOR.join(["sum_metric", "US"]): [6.0, 14.0],
+                FLAT_COLUMN_SEPARATOR.join(["sum_metric", "UK"]): [5, 12],
```

Comment:
NIce! I'm glad to see integers remain integers when summed. Would you mind updating the comment above as it still references floats.

Reply:
Done.